src/site/content/en/blog/codelab-stories-component/index.md

```diff
@@ -209,11 +312,32 @@ Last, we're using Javascript to toggle between our stories, why not give CSS a l
 }
 ```
 
-A keen eye would notice the `pointer-events: none` there and scratch their head. I'd say this is the only downside of the solution so far, is that a `.seen.story` is still on top and will receive taps, even though it's invisible. By setting the `pointer-events` to none, we turn the glass story into a window, and steal more user interactions. Not too bad of a trade off, not too hard to manage here in our CSS right now. We're not juggling `z-index`, I'm feeling good about this still.
+The `.seen` class will be added to a story that needs an exit.
+I got the custom easing function (`cubic-bezier(0.4, 0.0, 1,1)`)
+from Material Design's [Easing](https://material.io/design/motion/speed.html#easing)
+guide (see the *Accerlerated easing* section).
+
+{% Aside %}
+TODO(kayce): Explain the ampersand syntax
+{% endAside %}
 
+If you've got a keen eye you probably noticed the `pointer-events: none`
+declaration and are scratching your head right now. I'd say this is the only
+downside of the solution so far. We need this because a `.seen.story` element
+will be on top and will receive taps, even though it's invisible. By setting the
+`pointer-events` to `none`, we turn the glass story into a window, and steal
+more user interactions. Not too bad of a trade off, not too hard to manage here
+in our CSS right now. We're not juggling `z-index`. I'm feeling good about this
+still.
 
-## Javascript
-The interactions of a stories component are quite simple to the user: tap on the right to go forward, tap on the left to go back. Simple things for users tends to be hard work for developers. We'll take care of lots of it though, it'll get us really far.
+TODO(kayce): We need to rephrase the last paragraph. Turning the glass story into
+a window? Stealing more user interactions? 
```

Comment:
```suggestion
```

I think the edit I made above solves this problem, thoughts? Sorry you had to inherit that confusing typo!

Reply:
Makes sense now